r2/r2/public/static/js/flair.js

```diff
@@ -221,3 +221,11 @@ $(function() {
 
     $(".flairselector .dropdown").click(toggleFlairSelector);
 });
+
+$.fn.remove_link_flair_class = function () {
+  $(this)
+    .removeClass("linkflair")
+    .attr('class', function(i, c) {
+      return (c.replace(/(^|\s)linkflair\S+/g, ''));
+    });
+};
```

Comment:
nit: see comment about about renaming this to be camelCased.

Also, would you mind moving this to `jquery.reddit.js`? I'd like to keep all of this type of function (custom jQuery extensions that exist solely because of our jquery-json api responses) all in one place.